robotsim.py

```diff
@@ -296,11 +398,27 @@ def __verifyPath(self,path): # path : ['L', 'R', 'U', 'D'...]
 
     def debugTile(self):
```

Comment:
Two functions that do almost the same seems redundant, but its okay as they are internal.

Reply:
yeiiii